sdk/search/Azure.Search.Documents/src/Generated/Models/AnalyzeTextOptions.cs

```diff
@@ -23,23 +24,8 @@ public AnalyzeTextOptions(string text)
             }
 
             Text = text;
-            TokenFilters = new List<TokenFilterName>();
-            CharFilters = new List<string>();
-        }
-
-        /// <summary> Initializes a new instance of AnalyzeTextOptions. </summary>
-        /// <param name="text"> The text to break into tokens. </param>
-        /// <param name="analyzerName"> The name of the analyzer to use to break the given text. If this parameter is not specified, you must specify a tokenizer instead. The tokenizer and analyzer parameters are mutually exclusive. </param>
-        /// <param name="tokenizerName"> The name of the tokenizer to use to break the given text. If this parameter is not specified, you must specify an analyzer instead. The tokenizer and analyzer parameters are mutually exclusive. </param>
-        /// <param name="tokenFilters"> An optional list of token filters to use when breaking the given text. This parameter can only be set when using the tokenizer parameter. </param>
-        /// <param name="charFilters"> An optional list of character filters to use when breaking the given text. This parameter can only be set when using the tokenizer parameter. </param>
-        internal AnalyzeTextOptions(string text, LexicalAnalyzerName? analyzerName, LexicalTokenizerName? tokenizerName, IList<TokenFilterName> tokenFilters, IList<string> charFilters)
-        {
-            Text = text;
-            AnalyzerName = analyzerName;
-            TokenizerName = tokenizerName;
-            TokenFilters = tokenFilters ?? new List<TokenFilterName>();
-            CharFilters = charFilters ?? new List<string>();
+            TokenFilters = new ChangeTrackingList<TokenFilterName>();
+            CharFilters = new ChangeTrackingList<string>();
```

Comment:
If this corresponds to `CharFilterName`, it isn't using it, thus reinforcing that type shouldn't be added (no use).

Reply:
It's not using the type because the property is redefined by customization code.

Reply:
Since this AutoRest.CSharp update is adding it, I believe you should hide it since it's unused and "new". It was never autogenerated before.